bellatrix.core/src/main/java/solutions/bellatrix/core/utilities/HtmlService.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2024 Automate The Planet Ltd.
+ * Author: Miriam Kyoseva
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package solutions.bellatrix.core.utilities;
+
+import lombok.experimental.UtilityClass;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Element;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Web Utility Class
+ */
+@UtilityClass
+public class HtmlService {
+    public static String findCssLocator(String html, String xpath) {
```

Comment:
The name of the method is ` findCssLocator` but I need to pass xpath?

Reply:
Tried renaming those methods to something less confusing - convertXpathToAbsoluteCssLocator.

Do suggest better naming